Services/DataBaseService/Mappers/UserMapper.cs

```diff
@@ -0,0 +1,20 @@
+﻿using DataBaseService.DbModels;
+using DataBaseService.Interfaces;
+using DTO;
+using System;
+
+namespace DataBaseService.Mappers
+{
+    public class UserMapper : IMapper<UserEmailPassword, DbUser>
+    {
+        public DbUser CreateMap(UserEmailPassword data)
+        {
+            return new DbUser { Id = Guid.NewGuid(), Email = data.Email, Password = data.Password };
```

Comment:
We don't check id in the database? Whether it already exists? I know that it is almost impossible, but it is the reason why we set it in the database.  We can't let take one user money to another by not correct Id

Reply:
This is one of the reasons why we choose guid. We don`t need to check this value in database

Reply:
It was made as GUID because of security : to someone hasn't catch request. And IF somehow it would be the same for two persons? Don't u think that better check?

Reply:
If we used Guid.NewGuid(), we don`t need to check (see Spartak`s answer)